test/unit/master/test_subjob.py

```diff
@@ -36,6 +36,12 @@ def setUp(self):
             ],
         )
 
+    def test_set_atoms_subjob_id(self):
+        atoms = [Mock(), Mock()]
+        self._subjob._set_atoms_subjob_id(atoms, 4)
+        for atom in atoms:
+            self.assertEqual(atom.subjob_id, 4)
+
```

Comment:
Also, it would be great if we keep with the convention of naming tests like "test_X_does_Y_when_Z" or something similar -- basically including a description of the tested behavior in the test name.

I prefer to name tests that way because it helps us think about tests as a "spec" for the application. If we just name our tests "test_<method_name>" then it is not clear what exactly we're expecting the test to do, and it is tempting for that test to just expand to test *everything* that the method does instead of splitting tests into behavioral units.